android/build.gradle

```
@@ -7,8 +7,10 @@ buildscript {
             google()
             jcenter()
         }
+        def buildGradleVersion = ext.has('buildGradlePluginVersion') ? ext.get('buildGradlePluginVersion') : '4.1.2'
```

```suggestion
        def buildGradleVersion = ext.has('buildGradlePluginVersion') ? ext.get('buildGradlePluginVersion') : '4.1.1'
```

This was my mistake, current version is still 4.1.1, there is not (and probably will not be?) a 4.1.2 - I'll suggest this but then also commit it so this can move forward, sorry